frontend/src/pages/modelServing/screens/projects/ServingRuntimeDetails.tsx

```diff
@@ -22,7 +22,7 @@ const ServingRuntimeDetails: React.FC<ServingRuntimeDetailsProps> = ({ obj }) =>
   const container = obj.spec.containers[0]; // can we assume the first container?
   const sizes = getServingRuntimeSizes(dashboardConfig);
   const size = sizes.find((size) => _.isEqual(size.resources, container.resources));
-  const [accelerator] = useServingRuntimeAccelerator(obj);
+  const [accelerator] = useServingAccelerator(obj);
```

Comment:
hah -- FWIW, the naming change was not needed... `useServingRuntimeAccelerator` was fine lol, I was just using the shorter name as an example.

Reply:
oh lol. i actually removed the whole useServingRuntimeAccelerator hook but then in a later commit added it back (with a different name) so to me i was creating a new hook not changing the name